### PR TITLE
feat(second): add second function

### PR DESCRIPTION
**Pull request by yeashinr**
Please let me merge.

### CODE REVIEW

File: src/second.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (kentcdodds) [inline src/second.js]: Could you remove the irrelevant comments in this file please?

File: test/second.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import test from 'ava'
+import {second} from '../src'
+
+test('Check Array contains negative value', t => {
```
Comment (kentcdodds) [inline test/second.test.js]: Could you update the title of this test please?

**Comment by yeashinr** [comment]
I have resolved the issues that you mentioned. Not sure about the conflicting issus. How to check the conflictec is resolved or not?

**Comment by yeashinr** [comment]
I have resolved the issues that you mentioned. But, not sure about the conflicting one. I rebase onto master. Not sure whether it solves the issue or not.

**Comment by kentcdodds** [comment]
There are still conflicts, but I can fix those for you no problem :) I'm not sure what happened.

**Pull request merged**